libs/zzz/consts/src/common.ts

```diff
@@ -133,9 +134,15 @@ export const statKeyTextMap: Partial<Record<string, string>> = {
   def_mult_: 'DEF Multiplier',
   res_mult_: 'Resistance Multiplier',
   base: 'Base Damage',
+  enemyAnomBuildupRes_mult_: 'Enemy Anomaly Buildup RES Multiplier',
+  anomBuildup_mult_: 'Anomaly Buildup Bonus Multiplier',
+  anomMas_mult_: 'Anomaly Mastery Bonus',
+  anomalyBuildupInst: 'Anomaly Buildup',
```

Comment:
I think this and dazeInst should be handled in charbase?
https://github.com/frzyc/genshin-optimizer/blob/71968753630489f2cb10a6bf200b6ebc6181345e/libs/zzz/formula-ui/src/char/CharBase.tsx#L60